security/pfSense-pkg-zeek/files/usr/local/pkg/zeek.inc

```diff
@@ -0,0 +1,982 @@
+<?php
+/*
+* zeek.inc
+* part of pfSense (https://www.pfSense.org/)
+* Copyright (c) 2018-2020 Prosper Doko
+* Copyright (c) 2020 Mark Overholser
+* All rights reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+require_once('config.inc');
+require_once('globals.inc');
+require_once('notices.inc');
+require_once('pfsense-utils.inc');
+require_once('pkg-utils.inc');
+require_once('services.inc');
+require_once('service-utils.inc');
+require_once("functions.inc");
+require_once('util.inc');
+require_once("interfaces.inc");
+require_once('xmlrpc_client.inc');
+
+if (!function_exists("filter_configure")) {
+	require_once("filter.inc");
+}
+
+$shortcut_section = "zeek";
+
+define('ZEEK_LOCALBASE', '/usr/local');
+define('ZEEK_ETC', ZEEK_LOCALBASE .'/etc');
+define('ZEEK_LOGDIR', ZEEK_LOCALBASE .'/logs/current');
+define('ZEEK_SITEDIR', ZEEK_LOCALBASE .'/share/zeek/site');
+
+/*
+* Utility functions
+*/
+
+/* Handle base64 encoding and linebreaks in textarea configuration fields*/
+function zeek_text_area_decode($text) {
+	return preg_replace('/\r\n/', "\n", base64_decode($text));
+}
+
+/* Get interface IP and netmask for Zeek interfaces*/
+function zeek_get_real_interface_address($iface) {
+	if (!function_exists("get_interface_ip")) {
+		require_once("interfaces.inc");
```

Comment:
There is already a `require_once()` for `interfaces.inc` at the start of this file, this is almost certainly unnecessary.